src/ui/src/sessioninfo.cpp

```diff
@@ -44,7 +45,8 @@ void SessionInfo::retrieveFromStorage( QSettings& settings )
             if ( settings.contains( "OpenFiles/version" ) ) {
                 settings.beginGroup( "OpenFiles" );
                 if ( settings.value( "version" ).toInt() == OPENFILES_VERSION ) {
-                    int size = settings.beginReadArray( "openFiles" );
+                    const int size = settings.beginReadArray( "openFiles" );
```

Comment:
Another settings place. Let's remove reserve

Reply:
fix it https://github.com/variar/klogg/commit/a4f9cad9eceff87a667f1c6cec939f383153a6e3